docarray/documents/mesh/vertices_and_faces.py

```diff
@@ -23,7 +23,7 @@ class VerticesAndFaces(BaseDoc):
     faces: AnyTensor
 
     @classmethod
-    def validate(
+    def _docarray_validate(
```

Comment:
why is this needed?

Reply:
this is an alias, the signture function of the validate function change between pydtnic v1 and v2, therefore I created this function to abstract away the validation logic in one place and then depending on the pydantic version install I translate it to the right function singature